oeis.py

```diff
@@ -119,6 +119,14 @@ def A000290(start: int = 0, limit: int = 20) -> Collection[int]:
 
     return sequence
 
+def A001462(n=6):
+    "Golomb's sequence: F(n) = 1 + F(n - F(n - 1))"
```

Comment:
Where did you find this definition? Something's wrong, you wrote `F(n) = 1 + F(n - F(n - 1))` but you implemented `F(n) = 1 + F(n - F(F((n - 1))))`, which one is the right one?